presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/TestPostgreSqlTypeMapping.java

```diff
@@ -265,6 +265,26 @@ public void testDate()
         }
     }
 
+    @Test
+    public void testEnum()
+    {
+        JdbcSqlExecutor jdbcSqlExecutor = new JdbcSqlExecutor(postgreSqlServer.getJdbcUrl());
+        jdbcSqlExecutor.execute("CREATE TYPE enum_t as enum ('a','b','c')");
+        jdbcSqlExecutor.execute("CREATE TABLE tpch.test_enum(id int, s varchar, e enum_t)");
+        jdbcSqlExecutor.execute("INSERT INTO tpch.test_enum(id,s,e) values (1,'a','a'::enum_t)");
+        try {
+            assertQuery("SELECT column_name, data_type FROM information_schema.columns WHERE table_schema = 'tpch' AND table_name = 'test_enum'",
+                                "VALUES ('id','integer'),('s','varchar'),('e','varchar')");
+            String expected = "VALUES (1,'a','a')";
```

Comment:
I think using only 1 character for both the column name and the value looks confusing in this case.